packages/gator-permissions-snap/src/utils/validate.ts

```diff
@@ -50,6 +52,7 @@ const zRevocationParams = z.object({
  */
 export function validateRevocationParams(params: Json): {
   permissionContext: Hex;
+  metadata?: RevocationMetadata;
```

Comment:
metadata should be required - it's required (although not validated) in `zRevocationParams`, but optional on the return type here.